### PR TITLE
Acceptance test buy sub

### PR DESCRIPTION
**Pull request by mario-galic**
@afiore @rtyley 

Buy subscription acceptance test (`CheckoutSpec.scala`) has been modified as follows:
* use test-users facility
* expose test-user secret as environmental variable in Travis
* set testing cookies: `ANALYTICS_OFF_KEY`, `pre-signin-test-user`
* set cookie `qa-passthrough` (Is this needed??)
* fix flakiness by waiting for last element in relevant subsection of the page to load before proceeding
* add `acceptance-test-buy-sub` testOnly SBT task, which executes just the buy subscription test
* configure Prout to execute just buy subscription test

The test itself (without build time) completes in around 30 sec.

I have disabled other acceptance tests temporarily to allow for easier monitoring of test stability. Once it seems stable we can gradually re-enable/add other acceptance tests.

Result of testing-in-production:
* [Travis build](https://travis-ci.org/guardian/subscriptions-frontend/builds/87668704)
* [Sauce Labs screencast](https://saucelabs.com/tests/66bac2af77fa47d1a28f68295f417196)
* [Zuora screenshot](https://github.com/guardian/subscriptions-frontend/files/21786/Screen.Shot.2015-10-27.at.14.10.42.pdf)
* [Salesforce screenshot](https://github.com/guardian/subscriptions-frontend/files/21787/Screen.Shot.2015-10-27.at.14.11.23.pdf)

This PR in combination with https://github.com/guardian/prout/pull/27 should implement an end-to-end automated acceptance test with test result feedback to developer as a comment in the original PR.

### CODE REVIEW

**Comment by rtyley** [comment]
:+1: from me, but glad to know what @afiore thinks!

File: test/acceptance/Util.scala [hunk 1]
```diff
@@ -30,14 +33,33 @@ trait Util { this: WebBrowser =>
     go.to(appUrl)
     driver.manage().deleteAllCookies()
     driver.manage().timeouts().implicitlyWait(60, TimeUnit.SECONDS)
+
+    addTestUserCookies()
+  }
+
+  private def addTestUserCookies() = {
+    val passthroughCookie = new Cookie(qaCookie.name, qaCookie.value)
```
Comment (afiore) [inline test/acceptance/Util.scala]: This is probably not needed as it was intended for staff-restricted pages

**Comment by afiore** [comment]
:+1: looking good! It probably would be good to re-enable the other tests sooner rather than later

**Comment by mario-galic** [comment]
Per comments, I removed the passthrough cookie, and enabled one more acceptance test.

Tests (against PROD) now complete within [35 s](https://travis-ci.org/guardian/subscriptions-frontend/builds/87871524)

It is still working: [Sauce Labs screencast](https://saucelabs.com/tests/7ad42d5510ba41acb97cbff9e0b1ee7a)

**Comment by mario-galic** [comment]
Note that to run the acceptance test locally you need to now expose the test user secret locally (see `test/conf/application.conf`). Test user secret is currently not in private `.conf` file.

**Pull request merged**